src/JavaCodeGenerator.cs

```diff
@@ -153,24 +154,36 @@ public override Task Generate(CodeModel codeModel)
                 javaFiles.Add(GetModelJavaFile(codeModel, javaSettings, modelType));
             }
 
-            foreach (ServiceEnum serviceEnum in service.Enums)
+            if (service.Enums != null && service.Enums.Any())
```

Comment:
Did you have issues where omitting the .Any() check produced bad behavior? Otherwise, it seems simpler to let the foreach loop iterate 0 times for an empty collection.

Reply:
No, I didn't have any issues. This was simply an example of a premature optimization. Thanks for pointing it out. I've removed it.